### PR TITLE
Space-travelers-hub

### PR DESCRIPTION
**Pull request by yodit93**
### In this project, we

- [ ] configured basic setup
- [ ] fetched data and update Redux store
- [ ] rendered UI:lists
- [ ] wrote actions and reducers for booking rockets and joining missions
- [ ] wrote actions and reducers for canceling rockets and leaving missions
- [ ] rendered UI: conditional components rendering
- [ ] rendered UI: My Profile section
- [ ] added tests

### CODE REVIEW

**Review by youmari — CHANGES_REQUESTED** [review]
Hi @yodit93 @Moses-chibuike , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
✔️ Clean Design 
✔️ PRO PR
✔️ No linter error

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/__test__/Rocket.test.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+});
+
+describe('Rockets', () => {
+  it('should render the Rockets component', () => {
+    const tree = renderer.create(<Provider store={store}><Rockets /></Provider>).toJSON();
+    expect(tree).toMatchSnapshot();
+  });
+  it('should have a list', () => {
+    render(
+      <Provider store={store}>
```
[the remark this answers is not in the document]
Comment (yodit93) [reply src/__test__/Rocket.test.js:32]: Thanks, I will check it.

**Review by ABDULALI3468 — CHANGES_REQUESTED** [review]
# Changes Required 👍 

Hi @yodit93, 



Indeed you did an exceptional but still, there is room for changes to complete this project. 💯


## Highlights 
✅State is updating according to requirements.
✅Design is well implemented.
✅All linters are checked.
✅Project is professionally documented.
✅The right GitHub flow method has been used.
✅Professional pull request title and description.


 ## Required Changes ♻️

> **_Check the comments below to have a look at the required changes._**

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

## Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification.

> * This is My **GitHub**: GitHub followers
> * This is My **Linkedin**: [Linkedin Badge](https://www.linkedin.com/in/abdul-ali-5400bb216/)&nbsp;
> * This is My **Twitter**: [Twitter Badge](https://twitter.com/mrabdul_ali)&nbsp;

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/Components/NavLink.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+          <li className="list-item">
+            <Link to="/profile">Profile</Link>
+          </li>
+        </ul>
+      </div>
+    </nav>
+  );
+}
+
+export default NavLink;
```
[the remark this answers is not in the document]
Comment (yodit93) [reply src/Components/NavLink.js:30]: Ok, I got it now, Thanks

**Review by GedeonTS — APPROVED** [review]
# STATUS: APPROVED :tada: :clap: :champagne: 
Hi Team @yodit93 and @Moses-chibuike 

You are amazing :smiley: :clap: !  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

 

## Highlights :clap: :tophat: 
- No linter errors :+1: 
- Good use of Gitflow :+1: 
- All tests are passing :+1: 
## Optional suggestions :small_airplane: 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/__test__/Rocket.test.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (yodit93) [reply src/__test__/Rocket.test.js:32]: I didn't update the snapshot after changing some syntax on the files. That is why it is falling.

**Comment by yodit93** [comment]
Thank you!!!

**Comment by Moses-chibuike** [comment]
> # STATUS: APPROVED 🎉 👏 🍾
> Hi Team @yodit93 and @Moses-chibuike
> 
> You are amazing 😃 👏 ! There is nothing else to say other than... it's time to merge it :shipit: Congratulations! 🎉
> 
>  [ 68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f42504a6d74685133595277443651716356442f67697068792e676966 ](https://camo.githubusercontent.com/395b3b21a125e95f2b70c16f35342677ad5350ec23bec5d6d9db1383f2c87427/68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f42504a6d74685133595277443651716356442f67697068792e676966) [ ](https://camo.githubusercontent.com/395b3b21a125e95f2b70c16f35342677ad5350ec23bec5d6d9db1383f2c87427/68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f42504a6d74685133595277443651716356442f67697068792e676966)
> 
> ## Highlights 👏 🎩
> * No linter errors 👍
> * Good use of Gitflow 👍
> * All tests are passing 👍
> 
> ## Optional suggestions 🛩️
> _Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._
> 
> Cheers and Happy coding!👏👏👏
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
> 
> _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Thanks for approving

**Pull request merged**